data/contactlist.json

```diff
@@ -1,45 +1,3 @@
 {
-  "persons" : [ {
-    "name" : "Alex Yeoh",
-    "email" : "alexyeoh@example.com",
-    "mod" : "CS2100",
-    "group" : "W13",
-    "tagged" : [ "friends" ],
-    "studentNumber" : "A1234567B"
-  }, {
-    "name" : "Bernice Yu",
-    "email" : "berniceyu@example.com",
-    "mod" : "CS2100",
-    "group" : "W12",
-    "tagged" : [ "colleagues", "friends" ],
-    "studentNumber" : "A1111111X"
-  }, {
-    "name" : "Charlotte Oliveiro",
-    "email" : "charlotte@example.com",
-    "mod" : "CS1101S",
-    "group" : "T06",
-    "tagged" : [ "neighbours" ],
-    "studentNumber" : "A2222222Y"
-  }, {
-    "name" : "David Li",
-    "email" : "lidavid@example.com",
-    "mod" : "PLS2001",
-    "group" : "T07",
-    "tagged" : [ "family" ],
-    "studentNumber" : "A3333333W"
-  }, {
-    "name" : "Irfan Ibrahim",
-    "email" : "irfan@example.com",
-    "mod" : "MA1101R",
-    "group" : "T08",
-    "tagged" : [ "classmates" ],
-    "studentNumber" : "A4444444H"
-  }, {
-    "name" : "Roy Balakrishnan",
-    "email" : "royb@example.com",
-    "mod" : "GER1000",
-    "group" : "T09",
-    "tagged" : [ "colleagues" ],
-    "studentNumber" : "A5555555Z"
-  } ]
+  "persons" : [ ]
```

Comment:
Let's exclude the contactlist.json commit from this PR! Its better to keep the sample data

Reply:
yeap sure